docs/plugins/algod.md

```diff
@@ -1,16 +1,17 @@
 # Algod Importer
 
-Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to
-provide old blocks.
+Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to provide old blocks.
```

Comment:
Not directly related to this PR but a suggested doc change: the algod links were changed very recently to drop `v2` in the docs:

```suggestion
Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod). The node must be configured as an archival node in order to provide old blocks.
```